gputop-webui/gputop-ui.js

```diff
@@ -985,11 +1009,11 @@ GputopUI.prototype.update_features = function(features) {
                 $("#" + metric.uuid).click(() => {
                     //$('#metrics-menu').addClass("active");
                     $('#metrics-tab-anchor').tab('show');
-                    this.select_metric_set(metric);
+                    this.select_metric_set(metric, this.current_hw_id);
                 });
             }).call(this, this.metrics_[i]);
         }
-        this.select_metric_set(this.metrics_[0]);
+        this.select_metric_set(this.metrics_[0], 0);
```

Comment:
Why not this.current_hw_id?

Reply:
yes, I have replaced this with this.current_hw_id.